pennylane_lightning/core/lightning_base.py

```diff
@@ -490,16 +491,35 @@ def loop(i, y):
 
         """
         # has jax dependency, so can't import up top
-        from .lightning_interpreter import LightningInterpreter
+        import jax
+        from pennylane.capture.primitives import AbstractMeasurement
 
-        if self.wires is None:
-            raise NotImplementedError("Wires must be specified for integration with plxpr capture.")
+        from .lightning_interpreter import LightningInterpreter
```

Comment:
@josephleekl here is the place where we would like to introduce more dependencies to support `jit` and `vmap`

Reply:
What dependency are we adding?

Reply:
@albi3ro from the context it reads like this file did not expect to be dependent on jax and `lightning_interpreter` seems the place to have it. Not sure if it's totally fine

Reply:
I'll reword the comment on 493.

Reply:
Mind elaborating? I'm not sure I follow.

Reply:
Thanks. Add some extra info there too.